src/core/instance/lifecycle.js

```diff
@@ -55,6 +57,29 @@ export function initLifecycle (vm: Component) {
   vm._isBeingDestroyed = false
 }
 
+let releaseStack = new WeakMap()
```

Comment:
we cannot use a weakmap as it would break support for IE

Reply:
Thanks for your code review, I had coding WeakMap in another way to support lower version IE.